gateway/src/apicast/policy.lua

```diff
@@ -13,7 +13,7 @@ local PHASES = {
     'rewrite', 'access',
     'content', 'balancer',
     'header_filter', 'body_filter',
-    'post_action',  'log'
+    'post_action',  'log', 'metrics',
```

Comment:
We should add new phases to the policies doc: https://github.com/3scale/apicast/blob/master/doc/policies.md#policies

Reply:
Done in https://github.com/3scale/apicast/pull/629/commits/c44b062f1047b718ac3a65773363708b1b1a704b